saleor/cart/forms.py

```diff
@@ -112,3 +113,11 @@ def save(self):
         product_variant = self.get_variant(self.cleaned_data)
         return self.cart.add(product_variant, self.cleaned_data['quantity'],
                              replace=True)
+
+
+class CountryForm(forms.Form):
+    country = forms.ChoiceField(choices=COUNTRY_CODE_CHOICES)
+
+    def get_shipment_options(self):
+        code = self.cleaned_data['country']
+        return ShippingMethodCountry.objects.filter(country_code=code)
```

Comment:
We should also support case, when there's no special pricing for particular country see: (https://github.com/mirumee/saleor/blob/master/saleor/shipping/models.py#L15)
According to the design, shipping options should be ordered by their price (cheapest first)